lua/plugins.lua

```diff
@@ -25,9 +25,6 @@ return {
     "nvim-telescope/telescope.nvim",
     config = function()
       require("core.telescope").setup()
```

Comment:
Also, it would be great if you could make this even more compact:
``` lua
config = require("core.telescope").setup
```

Reply:
Done